docs/api/products.rst

```diff
@@ -1,33 +1,168 @@
++===============+================+===================================================================+
+| Authorization | header         | Token received during authentication                              |
++---------------+----------------+-------------------------------------------------------------------+
+| page          | query          | *(optional)* Number of the page, by default = 1                   |
++---------------+----------------+-------------------------------------------------------------------+
+| limit         | query          | *(optional)* Number of items to display per page, by default = 10 |
++---------------+----------------+-------------------------------------------------------------------+
+
+
+Example
+.......
+
+.. code-block:: bash
+
+    curl http://sylius.dev/api/v1/products/
+        -H "Authorization: Bearer MWExMWM0NzE1NmUyZDgyZDJiMjEzMmFlMjQ4MzgwMmE4ZTkxYzM0YjdlN2U2YzliNDIyMTk1ZDhlNDYxYWE4Ng"
+        -H “Accept: application/json”
 
-To browse all products available in the store you should call the following GET request:
+Example Response
+~~~~~~~~~~~~~~~~
 
 .. code-block:: text
 
-    GET /api/v1/products/
+    STATUS: 200 OK
+
+.. code-block:: json
+
+    {
+        {
+	"page": 1,
+	"limit": 10,
+	"pages": 1,
+	"total": 1,
+	"_links": {
+		"self": {
+			"href": "/api/v1/products/?page=1&limit=10"
+		},
+		"first": {
+			"href": "/api/v1/products/?page=1&limit=10"
+		},
+		"last": {
+			"href": "/api/v1/products/?page=7&limit=10"
+		},
+		"next": {
+			"href": "/api/v1/products/?page=2&limit=10"
+		}
+	},
+	"_embedded": {
+		"items": [
+			{
+				"id": 61,
+				"code": "Banana",
+				"translations": {
+					"en_US": {
+                        "id": 61,
+						"locale": "en_US",
+						"name": "Banana",
+						"slug": "banana"
+					}
+				},
+				"_links": {
+					"self": {
+						"href": "/api/v1/products/61"
+					}
+				}
+			},
+        ]
+    }
 
-Parameters
-~~~~~~~~~~
+Getting a Single Product
+------------------------
+
+You can request detailed product information by executing the following request:
+
+Definition
+..........
+
+.. code-block:: text
+
+    GET /api/v1/products/{id}
+
++---------------+----------------+-------------------------------------------------------------------+
+| Parameter     | Parameter type | Description                                                       |
++===============+================+===================================================================+
+| Authorization | header         | Token received during authentication                              |
++---------------+----------------+-------------------------------------------------------------------+
+| id            | url attribute  | Id of requested resource                                          |
++---------------+----------------+-------------------------------------------------------------------+
+| page          | query          | *(optional)* Number of the page, by default = 1                   |
```

Comment:
This parameter and parameter below is not needed. We have this mistake in other parts of our documentation also.